AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/CodeGen/Data/HostedUIOptions.swift

```diff
@@ -8,8 +8,6 @@
 import Foundation
 import Amplify
 
-#if canImport(AuthenticationServices)
```

Comment:
Can this be wrapped around iOS and macOS too?

Reply:
I'll check if removing this doesn't cause excessive new compiler checks. Hopefully it wont 🤞